src/context.rs

```diff
@@ -1096,6 +1176,12 @@ impl CDFContext {
   }
 }
 
+impl fmt::Debug for CDFContext {
+  fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
@lu_zero, @rzumer, @negge : I remember that this mechanism for debugging CDFs are created with big efforts, do we still want to keep this instrument? It was originally requested by rzumer and negge, as I remember.

Reply:
I don't recall voicing an opinion about this, and I have no objection with removing it if it is deemed beneficial.